src/app/tools/page.tsx

```diff
@@ -0,0 +1,138 @@
+// pages/tools-and-useful-links.js
+
+import React from "react";
+
+const csenLinks = [
+  {
+    name: "B.S. in Computer Science and Engineering",
+    link: "https://www.scu.edu/engineering/academic-programs/department-of-computer-engineering/undergraduate/computer-science-and-engineering-major/",
+  },
+];
+
+const officeOfRegistrarLinks = [
```

Comment:
Should we keep these objects in a separate file? I think it's just good practice for readability, but since I don't see any other page accessing these objects I think it's fine.